.github/workflows/hello-reason.yml

```
@@ -93,14 +94,13 @@ jobs:
             "source": "./package.json",
             "override": {
               "dependencies": {
-                "reason-mobile": "github:EduardoRFS/reason-mobile:generate.json#'"${GITHUB_SHA}"'"
+                "${{ github.event.repository.name }}": "github:'"${GITHUB_REPOSITORY}"':generate.json#'"${GITHUB_SHA}"'"
```

```suggestion
                "reason-mobile": "github:'"${GITHUB_REPOSITORY}"':generate.json#'"${GITHUB_SHA}"'"
```

Same reasoning as above